tests/debug.test.js

```diff
@@ -50,4 +52,24 @@ describe('debugCommand()', () => {
         sinon.assert.calledWithMatch(fakeProcess.stdout.write, 'msgstr[1] "{0} :sƃɐʇ pıʅɐʌuI');
       });
   });
+
+  it('should support keepOriginalHeader', () => {
+    const writeFileMock = sinon.spy();
+
+    const poToolsModule = rewire('./node_modules/pofile/lib/po.js');
+
+    const revertMock = poToolsModule.__set__('fs', {
+      writeFile: writeFileMock,
+    });
+
+    return debugCommand({potFile: './tests/fixtures/source.pot', format: 'unicode', output: './tests/fixtures/output.po'}, {_chalk: fakeChalk, _process: fakeProcess, _console: fakeConsole})
```

Comment:
`potFile` should be `potfile`. Nothing happening is caused by the process exiting due to poLoad's error being caught. 

If you keep the assertion (as per the other tests here) that process.exit *isn't* called, that helps catch errors if the process is exiting due to a problem. 

If you wanted to remove the console stubbing then I think that would be ok too. I like test output to be quiet in general, but that can hide error messages as you've found.

I would avoid using rewire, it's not necessary. I'd use the same convention as these other tests and look at what's being written to the process stub. That will get your test up and running.